src/private/Draggable_p.h

```diff
@@ -85,6 +85,14 @@ class DOCKS_EXPORT Draggable
      * If true, means the drag will simply move the existing window, and no undocking/untabbing is involved.
      */
     virtual bool isWindow() const = 0;
+
+    /**
+     * @brief Maps the given point in draggable mouse area`s coordinate system to
+     * the equivalent point in window's coordinate system,
+     * and returns the mapped coordinate.
+     */
+    virtual QPoint mapToWindow(const QPoint& pos) const { return pos; };
```

Comment:
Did you forget to commit the place where you call this method

Reply:
thanks, fixed